app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -121,24 +120,30 @@ module.exports = React.createClass
   handleFocus: ->
     @menus[0].focus()
 
-  getOptions: (i, optionsKeys) ->
-    optionsKey = if optionsKeys.length then optionsKeys.join(';') else '*'
+  getOptionsKey: (i, optionsKey = '') ->
+    {selects} = @props.task
+    select = selects[i]
+    [condition] = selects.filter (filterSelect) => filterSelect.id is select.condition
+    conditionIndex = selects.indexOf(condition)
+    optionsKey = if optionsKey.length is 0 then @props.annotation.value[conditionIndex]?.value else "#{@props.annotation.value[conditionIndex]?.value};#{optionsKey}"
+    if condition.condition? then @getOptionsKey(conditionIndex, optionsKey) else optionsKey
```

Comment:
maybe `selectCondition` instead? not sure condition is the best word for something that is required before the select in focus. just did a quick Google thesaurus search and a few contender words popped up - antecedent or precursor?